### PR TITLE
Prevent thread blocking by adding sleep

### PR DESCRIPTION
**Pull request by anderscognite**
This increases loading time for IAA with 1 second, but does not freeze GUI during parsing which is very nice.

### CODE REVIEW

File: src/parsers/protobuf/main.ts [hunk 1]
```diff
@@ -95,15 +94,17 @@ export default function parseProtobuf(
   const treeIndexNodeIdMap: TreeIndexNodeIdMap = [];
   const colorMap: ColorMap = [];
 
-  const handleWebNode = (webNode: any) => {
-    const { boundingBox, path, id } = webNode;
+  let count = 0;
```
Comment (dragly) [inline src/parsers/protobuf/main.ts:97]: Should this maybe be inside the function to make the behavior deterministic also when we have multiple viewers/parses in the same project? Just makes it a bit easier to debug down the line.

**Review by dragly — APPROVED** [review]

File: src/parsers/protobuf/main.ts (hunk 1 above)
Comment (anderscognite) [reply src/parsers/protobuf/main.ts:97]: It _is_ inside the `parseProtobuf` function, so multiple viewers / parsers will have different counts.

**Comment by anderscognite** [comment]
Also note the change in the parseProtobuf API. This PR or separate?

File: src/parsers/protobuf/main.ts (hunk 1 above)
Comment (dragly) [reply src/parsers/protobuf/main.ts:97]: My bad :)

**Comment by dragly** [comment]
I think it is okay to keep in here, but the tests need to be updated.

**Comment by anderscognite** [comment]
Yes.

**Pull request merged**